aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -590,49 +725,42 @@ spec aptos_framework::account {
         ensures signer::address_of(result_1) == resource_addr;
         let post offer_for = global<Account>(resource_addr).signer_capability_offer.for;
         ensures option::spec_borrow(offer_for) == resource_addr;
-        ensures result_2 == SignerCapability { account: resource_addr };
+        ensures result_2 == SignerCapability {account: resource_addr};
     }
 
     /// Check if the bytes of the new address is 32.
     /// The Account does not exist under the new address before creating the account.
     /// The system reserved addresses is @0x1 / @0x2 / @0x3 / @0x4 / @0x5  / @0x6 / @0x7 / @0x8 / @0x9 / @0xa.
     spec create_framework_reserved_account(addr: address): (signer, SignerCapability) {
         aborts_if spec_is_framework_address(addr);
-        include CreateAccountAbortsIf {addr};
+        include CreateAccountAbortsIf { addr };
```

Comment:
Why does the tool add spaces here, which seems to be inconsistent with the case above where the spaces are removed?

Reply:
We will fix this.